MTA-6893 Document Hub OIDC authentication (#382)
migtools/mta-documentation: docs/topics/mta-install/ref_openldap-active-directory-configurations.adoc

:_newdoc-version: 2.18.5
:_template-generated: 2025-07-31
:_mod-docs-content-type: REFERENCE

[id="openldap-active-directory-configurations_{context}"]
= OpenLDAP and Active Directory configurations

[role="_abstract"]

> The OpenLDAP and Active Directory (AD) implementations have different configurations for select fields.

Reviewer: ```suggestion
OpenLDAP and Active Directory (AD) implementations require different configurations for specific fields.
```
